nixos/modules/config/fonts/fontconfig.nix

```diff
@@ -253,7 +253,7 @@ in
       fontconfig = {
         enable = mkOption {
           type = types.bool;
-          default = true;
+          default = false;
```

Comment:
I think this should be changed only for `stateVersion` ≥ 21.03.

Reply:
IIRC the consensus was to use `stateVersion` only for stateful things, not to delay breaking changes in NixOS options. It has the added benefit that any issues will be picked up be people running unstable as soon as possible.

Reply:
Aha, cool.